multicluster/cmd/link.go

```diff
@@ -428,7 +470,32 @@ func renderServiceMirror(values *multicluster.Values, valuesOverrides map[string
 		}
 	}
 
-	return out.Bytes(), nil
+	if format == "json" {
```

Comment:
Same thought as above, could this make use of the library function introduced in the previous PR?